test/interval_tests/loop_tests.jl

```diff
@@ -57,17 +57,50 @@ function calc_pi3(N)
     sqrt(6*S3)
 end
 
+function calc_pi4(N)
+    S4 = @floatinterval(0)
+    II = @floatinterval(1)
+
+    for i in N:-1:1
+        S4 += II / (i^2)
+    end
+    S4 += II / @floatinterval(N, N+1)
+
+    sqrt(6*S4)
+end
+
+function calc_pi5(N)
+    S5 = @floatinterval(0)
+    II = @floatinterval(1)
+
+    for i in N:-1:1
+        S5 += 1 // (i^2)
+    end
+    S5 += 1 / @floatinterval(N, N+1)
+
+    sqrt(6*S5)
+end
+
 
 facts("Pi tests") do
 
+    big_pi = with_bigfloat_precision(256) do
+        big(pi)
+    end
+
```

Comment:
Shouldn't we make the tests related to pi, precision independent? 

I think, it is better to carry on the tests defining `big_pi` (in the lines above) as
```julia
big_pi = with_bigfloat_precision(get_bigfloat_precision()) do
    big(pi)
end
```